CONTRIBUTING.md

```diff
@@ -18,9 +18,18 @@ All commits you make should adhere to our commit guidelines. We use [conventiona
 
 This project uses Prettier for code formatting. You can run `make format` to format all your code before you submit your code for review.
 
-## Deployment
+## Publishing
+
+This project uses GitHub Actions to publish the packages automatically to npm. Ne packages are published after new tag is pushed to main branch. PR can be merged only by appropriate group of maintainers.
+
+### Steps to create a new package version
+
+1. Merge all appropriate PRs you want to publish into main branch
+1. Run the `make version` command to bump the version number in packages (new version number is determined automatically based on commit history)
+1. Check that the version number is correct and everything looks good
+1. Run `git push && git push --tags` to push the changes to remote
```

Comment:
So the `make version` command also commits the changes? Or do we have to do it manually?